pages/windows/clip.md

```diff
@@ -5,8 +5,16 @@
 
 - Pipe command line output to the Windows clipboard:
 
-`{{dir}} | clip`
+`dir|clip`
 
 - Copy the contents of a file to the Windows clipboard:
 
-`clip < {{path/to/file}}`
+`clip<path/to/file.ext`
+
+- Copy text with trailing newline to the Windows clipboard:
+
+`echo some text|clip`
```

Comment:
Perhaps this will read better?
```suggestion
`echo "{{some text}}" | clip`
```

Reply:
I'll add the curly brackets, however, echoing with quotes prints the quotes as well, which is not the intended result.